src/main/kotlin/extensions/yuan/varenyzc/BuildFromJsonObjectSupport.kt

```diff
@@ -0,0 +1,43 @@
+package extensions.yuan.varenyzc
+
+import extensions.Extension
+import wu.seal.jsontokotlin.model.builder.CodeBuilderConfig
+import wu.seal.jsontokotlin.model.builder.KotlinCodeBuilder
+import wu.seal.jsontokotlin.model.classscodestruct.KotlinClass
+import wu.seal.jsontokotlin.ui.jCheckBox
+import wu.seal.jsontokotlin.ui.jHorizontalLinearLayout
+import javax.swing.JPanel
+
+object BuildFromJsonObjectSupport : Extension() {
+
+    const val configKey = "top.varenyzc.build_from_json_enable"
+
+    override fun createUI(): JPanel {
+        return jHorizontalLinearLayout {
+            jCheckBox(
+                "Enable build from JsonObject",
```

Comment:
How about named it as ```add `fromJson` Function ```? It's not clear what is `Enable build from JsonObject`
Also add a small doc for this option can make people understand this feature fast as I think, you could reference https://github.com/wuseal/JsonToKotlinClass/blob/aabaa6a86bb5bb6d83b737904d285331c277ad91/src/main/kotlin/extensions/jose/han/ParcelableAnnotationSupport.kt#L27 to adding a Link